deps/v8/src/arm/assembler-arm.cc

```diff
@@ -50,6 +50,35 @@ bool CpuFeatures::initialized_ = false;
 unsigned CpuFeatures::supported_ = 0;
 unsigned CpuFeatures::found_by_runtime_probing_ = 0;
 
+#ifdef __arm__
+
+bool OS::ArmCpuHasFeature(CpuFeature feature) {
```

Comment:
I've screwed up this part of the patch... These changes should go in deps/v8/src/platform-freebsd.cc